client/src/pages/pot-instance/components/PotView/PotViewSwitcher.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Paper, ToggleButton, ToggleButtonGroup } from '@mui/material';
+import PotViewState from './PotViewState';
+import styles from './styles.module.css';
+
+interface Props {
+  selectedValue: PotViewState;
+  onChange: (value: keyof typeof PotViewState) => void;
+}
+
+const PotViewSwitcher = ({ selectedValue, onChange }: Props) => {
+  const handleChange = (
+    event: React.MouseEvent<HTMLElement>,
+    newValue: keyof typeof PotViewState
```

Comment:
keyof typeof?

Reply:
Forgot to remove those, playing around with enum types.